### PR TITLE
MileStone: 7 Mobile menu

### PR DESCRIPTION
**Pull request by developerwaleed**
#MileStone: 7 Mobile menu

>Add Close Button.
>Add Events to buttons.

### CODE REVIEW

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -6,32 +6,20 @@ env:
...
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v1
         with:
           node-version: "12.x"
-      - name: Setup Lighthouse
-        run: npm install -g @lhci/cli@0.7.x
-      - name: Lighthouse Report
-        run: lhci autorun --upload.target=temporary-public-storage --collect.staticDistDir=.
-  webhint:
-    name: Webhint
```
Comment (Wes-Isaac) [inline .github/workflows/linters.yml]: - [x] Great job so far, one of the project requirements is to add [JavaScript + HTML + CSS linter](https://github.com/microverseinc/linters-config/tree/master/html-css-js)., kindly fix that.

**Review by Wes-Isaac — CHANGES_REQUESTED** [review]
Hi @caasperr, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Project looks great ✔️.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: .github/workflows/linters.yml (hunk 1 above)
Comment (developerwaleed) [reply .github/workflows/linters.yml]: Thankyou @Wes-Isaac  Its fixed now! 👍

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @caasperr, and @teckim 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**